libbpf-tools/oomkill.c

```diff
@@ -78,11 +78,11 @@ static void handle_event(void *ctx, int cpu, void *data, __u32 data_sz)
 	strftime(ts, sizeof(ts), "%H:%M:%S", tm);
 
 	if (n)
-		printf("%s Triggered by PID %d (\"%s\"), OOM kill of PID %d (\"%s\"), %lld pages, loadavg: %s\n",
+		printf("%s Triggered by PID %d (\"%s\"), OOM kill of PID %d (\"%s\"), %lld pages, loadavg: %s",
```

Comment:
why removing '\n' here? The else branch still has '\n'. it seems '\n' makes sense since different oom's will be in different lines?

Reply:
The if branch print `/proc/loadavg` which has a newline already.